static/templates/settings/account-settings.handlebars

```diff
@@ -57,7 +57,7 @@
                             </button>
                         </a>
                         <i class="fa fa-question-circle change_name_tooltip settings-info-icon" data-toggle="tooltip"
-                          {{#unless page_params.is_admin}}{{#if page_params.realm_name_changes_disabled}}{{else}}{{#unless page_params.server_name_changes_disabled}}style="display:none"{{/unless}}{{/if}}{{/unless}}
+                          {{#if page_params.is_admin}}style="display:none"{{else}}{{#unless page_params.realm_name_changes_disabled}}{{#unless page_params.server_name_changes_disabled}}style="display:none"{{/unless}}{{/unless}}{{/if}}
```

Comment:
We recently merge support for doing this sort of conditional with a more natural and/or syntax:

`{{#if (and push_notifications_tooltip (not realm_push_notifications_enabled))}}disabled="disabled"{{/if}}`

See `settings_checkbox.hbs` for an example; can you migrate?

Reply:
@timabbott Done; there's a bunch of other places in this file that weirdly combine if/unless syntax that could use migration as well, might be worth opening an issue to keep track of all the Handlebars files that need logic refactoring.

Reply:
@andersk do you have plans to bulk-migrate the old syntax?

Reply:
I pushed a commit (totally untested) that moves this logic out of handlebars and into JavaScript; 

@synicalsyntax can you test that?

Also, I'm not sure we should need an `if true` -- do the handlebars docs recommend that as the way to eat whitespace?

Reply:
> @andersk do you have plans to bulk-migrate the old syntax?

Not at the moment.

> Also, I'm not sure we should need an `if true` -- do the handlebars docs recommend that as the way to eat whitespace?

You don’t need that, you can use comments instead.
* Eat left whitespace: `{{~!}}`
* Eat right whitespace: `{{!~}}`
* Eat left and right whitespace: `{{~!~}}`

Reply:
@timabbott The commit you added works, but the `{{~!~}}` syntax @andersk suggested doesn't work.

Reply:
Merged this under the theory we can clean this up forward, but did you try doing `{{~!~}}` both before and after the block?

Reply:
> The commit you added works, but the `{{~!~}}` syntax @andersk suggested doesn't work for stripping the whitespace.

It does strip whitespace. Perhaps your test showed otherwise because your commit doesn’t work either—it strips whitespace from the wrong place. The whitespace you’d need to strip is between `</button>` and `</a>`.

But isn’t the real problem here that `<button>` inside `<a>` is invalid HTML?

Reply:
> It does strip whitespace. Perhaps your test showed otherwise because your commit doesn’t work either—you’re stripping whitespace from the wrong place.

Thanks for the pointer, will fix with the suggested syntax in a followup PR.

> But isn’t the real problem here that <button> inside <a> is invalid HTML?
Yes, there's #6126 opened to fix that (and it hasn't really been acted upon even though it's probably a big concern). 

Speaking of buttons inside anchor links,
```handlebars
                        <a id="change_full_name" {{#if user_can_change_name}}href="#change_email"{{/if}}>
```
It looks like ce46cd914a615 changed `#change_full_name` to `#change_email` and the regression hasn't been fixed since because the full name modal still somehow triggers correctly (likely a side effect of some JS working against the HTML). I'll take a shot at #6126 to eliminate both of these issues.